scripts/pages/zoning/zoning.ts

```diff
@@ -444,10 +454,26 @@ class ZoneMenuHandler {
 				this.selectedZone.globalRegion?.type == null ||
 				this.selectedZone.globalRegion.type === GlobalRegionType.TYPES_LIST
 		);
+
+		// Restore scroll position recorded before lists were rebuilt
+		for (const [position, panel] of Object.entries(this.panels.selectionLists)) {
+			const scrollParent = panel?.GetParent();
```

Comment:
AFAIK a panel always has a parent, so this check seems unnecessary.

Reply:
Seems like can safely remove that check and `?.`, aren't the selection lists non-null anywhere?

Type definition for `GetParent` has it possibly returning null since technically the outermost panels of base layouts don't have parents. Could update the definition to not include that and leave a note in jsdoc, don't really mind. Otherwise would need to do `GetParent()!` to make TS happy in strict-mode.